examples/high_level_functions_basic_objects.js

```diff
@@ -0,0 +1,253 @@
+	lineForWeldedJoint.WeldedJoints([[lineWeldedJoint.no, surfaceForWeldedJoint.no, surface2ForWeldedJoint.no, undefined]]);
+	// Other
+	var line8 = new Line();
+	line8.RectangularPolygon(undefined, [nodes[114].coordinate_1, nodes[114].coordinate_2, nodes[114].coordinate_3], 2, 4);
+	line8.nPolygon(undefined, [nodes[149].coordinate_1, nodes[149].coordinate_2, nodes[149].coordinate_3], 5, 2);
+	/*********************************************************************************************/
+	
+	/*************************************** Surfaces ********************************************/
+	// Default plane surfaces
+	var surface = new Surface();
+	surface.Standard(undefined, linesForSurfaces[1][0], 1, "With default plane geometry type");
+	surface.WithoutThickness(undefined, linesForSurfaces[2][0], "With default plane geometry type");
+	surface.Rigid(undefined, linesForSurfaces[3][0], "With default plane geometry type");
+	surface.Membrane(undefined, linesForSurfaces[4][0], 1, "With default plane geometry type");
+	surface.WithoutMembraneTension(undefined, linesForSurfaces[5][0], 1, "With default plane geometry type");
+	surface.LoadTransfer(undefined, linesForSurfaces[6][0], "With default plane geometry type");
+	// Geometry type: quadrangle
+	nodes[202].coordinate_3 = 0.5;
+	surface.Standard(undefined, linesForSurfaces[7][0], 1);
+	surface.Quadrangle();
+	// Geometry type NURBS
+	var coreSurface = surface.Standard(undefined, linesForSurfaces[8][0], 1);
+	for (var i = 0; i < coreSurface.boundary_lines.length; ++i) {
+		coreSurface.boundary_lines[i].type = lines.TYPE_NURBS;
+	}
+	surface.NURBS();
+	// Geometry type: rotated
+	surface.Standard(undefined, linesForSurfaces[9][0], 1);
+	// TODO
+	// Geometry type: pipe
+	surface.Standard(undefined, linesForSurfaces[10][0], 1);
+	// TODO
+	// Option: grid for results
+	// TODO
+	// Option: hinges
+	surfaceWithHingesNo = surface.Standard(undefined, linesForSurfaces[11][0], 1).no;
+	var lineHinge = new LineHinge(undefined, undefined);
+	surface.Hinges([[surfaces[surfaceWithHingesNo].boundary_lines[1], lineHinge.no]]);
+	// Option: support
+	var surfaceSupport = new SurfaceSupport();
+	surfaceSupport.Fixed();
+	surface.Rigid(undefined, linesForSurfaces[12][0]);
+	surface.Support(1);
+	// Option: eccentricity
+	var surfaceEccentricity = new SurfaceEccentricity();
+	surface.Standard(undefined, linesForSurfaces[13][0]);
+	surface.Eccentricity(1);
+	// Option: mesh refinement (free meshing type)
+	var meshRefinement = new SurfaceMeshRefinement();
+	surface.WithoutThickness(undefined, linesForSurfaces[14][0]);
+	surface.MeshRefinement(1, 3);
+	// Option: specific axes
+	surface.Standard(undefined, linesForSurfaces[15][0]);
+	var inputAxesValues = [1, [45, undefined, undefined]];
+	surface.SpecificAxes(inputAxesValues);	// Angular rotation category
+	surface.Standard(undefined, linesForSurfaces[16][0]);
+	inputAxesValues = [2, [[surface.surface.boundary_lines[0], surface.surface.boundary_lines[2]], "Axis y"]];
```

Comment:
The lines defining the local coordinate system rotation are not connected. -> invalid object